README.rst

```diff
@@ -1,31 +1,33 @@
-xonsh
-=====
```

Comment:
It probably makes sens to keep the top-level title

Reply:
First word of the page is `<b>xonsh</b>`. Why we need additional header?

Reply:
I think most documents need titles.  The bold first word is just a nice bonus :wink:

Reply:
May be... for documents... but for people it just looks like visual noise here:
![Screenshot_20201019_230632](https://user-images.githubusercontent.com/1708680/96506121-c74f7b00-125f-11eb-8755-c7794922fd08.png)